Puppetfile.lock

```diff
@@ -178,16 +178,6 @@ GIT
       puppetlabs-stdlib (>= 2.2.1)
 
 GIT
-  remote: https://github.com/puppetlabs/puppetlabs-postgresql
```

Comment:
didn't notice that! will restore back.
was trying to get librarian-puppet update going must have removed that while experimenting.